### DISCUSSION TITLE
How to assign the base position of the robot?

### DISCUSSION THREAD

**Discussion by linhy97** [NVlabs/curobo#356] [Software Q&A] [answered]
Hi, when i use `MotionGenConfig.load_from_robot_config()` to load a robot (e.g. ur10) from a yml file, it seems no parameters to assign the location. 
Thus, i have some confuse:
(1) is the default position is to place the center point of robot's base_link as coordinate origin [0,0,0]
(2) If i want to change the position of robot, is there any method to set parameters?
Hope for your answer~

**Comment by Sepirence** [comment]
from helper import add_robot_to_scene
This method has position argument!

**Reply by linhy97** [reply]
Thank you for your help.
I find the helper.py in path `examples/isaac_sim/helper.py`, it is an isaac example. 
If i want to use in cuRobo, is it workable to add an robot to World?

**Comment by balakumar-s** [comment, marked as answer]
cuRobo represents everything with respect to the robot base frame. So you would have to convert from any additional offset from world to robot base frame external to the motion planning and other APIs. We provide a frame transformation helper here: https://curobo.org/_api/curobo.types.math.html#curobo.types.math.Pose

**Comment by shockstove** [comment]
May I ask how you answered the two questions you raised? Is the setting of robot coordinates using the function compute_local_pose? How is it used?
If you could give the answer, I would be extremely grateful.
